gui.py

```diff
@@ -0,0 +1,481 @@
+import time
```

Comment:
It will be a good habit to include copyright text, e.g., https://github.com/solvcon/modmesh/blob/42edba153b460845014ca44c47b601e6168174e2/modmesh/__init__.py#L1